macros/multiple_databases/dateadd.sql

```diff
@@ -0,0 +1,22 @@
+{%- macro dateadd(datepart, number, datetime_field) -%}
+{%- if target.type == 'snowflake' -%}
+    dateadd({{ datepart }}, {{ number }}, try_to_timestamp(to_varchar({{ datetime_field }})))
+{%- elif target.type == 'sqlserver' -%}
+    dateadd({{ datepart }}, {{ number }}, try_convert(datetime2, {{ datetime_field }}))
+{%- elif target.type == 'databricks' -%}
+    {%- set clock_component -%}
```

Comment:
can we rename this value to time_value? It represents the time only part of the datetime